src/bundleList.js

```diff
@@ -41,6 +41,7 @@ const venueRoles = [
   'venue'
 ];
 
+const SQLITE_REGEX = /whosonfirst-data-[a-z-]+.db/;
```

Comment:
Should this regex be able to match any WOF sqlite db?

```js
/whosonfirst-data-[a-z-]+.db/.test('https://dist.whosonfirst.org/sqlite/whosonfirst-data-constituency-us-ct-1481486175.db')

fals
```

Reply:
Nice catch ! This one will not work :disappointed: I change the regex.